browser/BUILD.gn

```diff
@@ -385,7 +385,15 @@ source_set("browser_process") {
       "brave_drm_tab_helper.cc",
       "brave_drm_tab_helper.h",
     ]
-    deps += [ "//brave/browser/widevine" ]
+    deps += [
+      "//base",
+      "//brave/browser/widevine",
+      "//brave/common:pref_names",
```

Comment:
not sure we need all of these. This could make sense if we moved `brave_drm_tab_helper.*` to a separate dir under `brave/`

Reply:
I see. will revert this changes.